### PR TITLE
ENH: natively support Graph in all Geary statistics

### PR DESCRIPTION
**Pull request by martinfleis**
~This one includes a fallback to W in `Geary_Local_MV` due to complicated refactoring that would need to happen in `_crand`.~ not anymore

### CODE REVIEW

**Review by sjsrey — APPROVED** [review]

**Comment by jGaboardi** [comment]
@martinfleis Is this still a draft or ready for review?

**Comment by martinfleis** [comment]
@jGaboardi it is ready for review, same as all the other PRs here but marked as draft as it currently needs one or two PRs merged to lib to pass at least with dev. I'd like to see the actual tests passing here before merging this.

File: esda/geary.py
Comment (jGaboardi) [inline esda/geary.py:109]:
```suggestion
                "w must be a libpysal.weights.W or libpysal.graph.Graph object, "
```

**Review by jGaboardi — APPROVED** [review]

File: esda/geary_local_mv.py
Comment (sjsrey) [inline esda/geary_local_mv.py:189]: This is a nice illustration of gains from using the graph over W, in terms of simplification.

**Review by sjsrey — APPROVED** [review]

**Comment by knaaptime** [comment]
i think we're good to merge all of these, then cut releases for lib and esda. Or would you prefer to see the lib changes propagate through the CI

**Comment by martinfleis** [comment]
I would probably cut lib and pin it here first.

**Comment by jGaboardi** [comment]
After merging https://github.com/pysal/libpysal/pull/740, let's cut the release of `libpysal`? Give me a thumb up here and I will merge that PR.

**Comment by knaaptime** [comment]
ok (also need to specify the pin in the readme)

**Comment by martinfleis** [comment]
Shall we maybe wait for https://github.com/pysal/libpysal/pull/733 to be finished? And then move Graph from experimental zone before a release?

**Comment by jGaboardi** [comment]
* CI is all green - `libpysal` is pinned - we have approvals
* I'll merge tomorrow in the AM unless someone else already has or there are detractors

**Pull request merged**
